### PR TITLE
Implement Codacy Recommendations

### PR DESCRIPTION
**Pull request by confused-Techie**
Alright, apologies to @DeeDeeG for the fact I just don't seem to stop making PPM PRs lol.

This PR should be the last one of the ones I've had semi ready to go, and implements the majority of the low hanging fruit flagged on codacy.

Thanks again to anyone able to review this one, and hopefully I'll calm down in this repo afterwards

### CODE REVIEW

File: script/download-node.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const downloadFileToLocation = function(url, filename, callback) {
   stream.on('error', callback);
   const requestStream = request.get(url)
   requestStream.on('response', function(response) {
-    if (response.statusCode == 404) {
+    if (response.statusCode === 404) {
```
Comment (DeeDeeG) [inline script/download-node.js:28]: "For fun" (out of curiosity), I checked that the `statusCode` is in fact a number, and not a string, when the URL is wrong. (I deliberately edited the hard-coded target URL to be wrong, logged the `typeof response.statusCode` here, and I got:

`typeof response.statusCode is: number`

Script behaves as expected with `=== 404` as opposed to `=== "404"` which bugs out.

Overly thorough way of saying this change is good 👍

**Review by DeeDeeG — APPROVED** [review]
Sure 👍 .

Tests pass.

Diff looks good.

**Comment by DeeDeeG** [comment]
Suprised how many unused `require()`s there are in this repo. Must have been a fair amount of churn throughout development.

**Comment by DeeDeeG** [comment]
> Alright, apologies to @DeeDeeG for the fact I just don't seem to stop making PPM PRs lol.

Lol, no worries. I happen to be available. Good to see some, er, winter cleaning around this repo, heh.

File: script/download-node.js (hunk 1 above)
Comment (confused-Techie) [reply script/download-node.js:28]: Thanks a ton for double checking! My intuition in this space said they should be numbers and I didn't question it. So I'm happy you've actually checked

**Comment by confused-Techie** [comment]
Thanks for the review @DeeDeeG!

And yeah it's surprising how much junk is left over in this repo, so I'm more than happy to just delete it lol

**Pull request merged**